modules/blockexplorer/blockexplorer_test.go

```diff
@@ -38,18 +39,12 @@ type explorerTester struct {
 func (et *explorerTester) csUpdateWait() {
 	<-et.csUpdateChan
 	<-et.beUpdateChan
-	et.tpUpdateWait()
-}
-
-// tpUpdateWait blocks until a transaction pool update has propagated to all
-// modules.
-func (et *explorerTester) tpUpdateWait() {
```

Comment:
getting rid of tpUpdateWait is a little weird because all testers have it. You're also probably going to want to have it if you start building custom blocks at some point.